e2e/provision/playbooks/roles/install/tasks/main.yml

```diff
@@ -12,10 +12,24 @@
   ansible.builtin.include_role:
     name: kpt
   loop:
-    - {pkg: nephio-system, version: nephio-system/v6, namespaces: [config-management-monitoring, config-management-system, nephio-system, porch-fn-system, porch-system, resource-group-system]}
-    - {pkg: nephio-webui, version: nephio-webui/v6, namespaces: [nephio-webui]}
+    - {pkg: porch-dev, version: porch-dev/v1, namespaces: [porch-fn-system, porch-system]}
+    - {pkg: nephio-controllers, version: nephio-controllers/v1, namespaces: [nephio-system]}
+    - {pkg: configsync, version: configsync/v1, namespaces: [config-management-monitoring, config-management-system, resource-group-system]}
```

Comment:
I was planning to extract some of this values as configurable parameters. I did some of that work in this [PR](https://github.com/nephio-project/test-infra/pull/47/files#diff-53d3c7ea5d1ef36dc93d1c69c492c135a49807b16973161d0f69e393926663df), more likely for R1+

Reply:
ok. I want to split out the "base install" vs the "sandbox install". I think it's close now but now quite there.